src/wasm-interpreter.h

```diff
@@ -129,13 +127,16 @@ class Indenter {
 template<typename SubType>
 class ExpressionRunner : public OverriddenVisitor<SubType, Flow> {
 protected:
-  // Keep a record of call depth, to guard against excessive recursion.
-  size_t depth = 0;
+  Index maxDepth;
+
```

Comment:
nit: this whitespace looks unnecessary.

Reply:
It's sort of separating the input values from the internal state values. Is there a clearer way to do that? I could use another `protected:` block lower down I guess.

Reply:
Nope, this newline seems fine if it's there intentionally :+1: